docs/source/programming_resources/tutorial_specific/blocks/controlling_a_servo/Controlling-a-Servo-(Blocks).rst

```diff
@@ -1,5 +1,5 @@
-Controlling a Servo (Blocks)
-----------------------------
+Controlling a Servo :bdg-warning:`Blocks`
+==========================================
```

Comment:
Let's make headings tags equal to the content above them. I know the specification says "at least as long" but we have thus far been really good about "exactly the length".